### PR TITLE
[BH-1672] Add logging of eMMC parameters

### PR DESCRIPTION
**Pull request by Lefucjusz**
Added logging of eMMC storage card
parameters so that it's easy to
determine what chip is installed
in the device the logs are from.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

**Review by dawidwojtas-mudita — APPROVED** [review]

**Review by dawidwojtas-mudita — APPROVED** [review]
LGTM :)

**Review by swichu91 — APPROVED** [review]

**Review by SP2FET — APPROVED** [review]

File: module-platform/rt1051/src/disk_emmc.hpp [hunk 1]
```diff
@@ -35,17 +36,22 @@ namespace purefs::blkdev
...
         pm_state pmState{pm_state::active};
         mutable cpp_freertos::MutexRecursive mutex;
         std::atomic<hwpart_t> currHwPart{0};
 
         std::unique_ptr<_mmc_card> mmcCard;
         std::shared_ptr<drivers::DriverUSDHC> driverUSDHC;
+
+        /* Read from datasheets */
+        const std::map<std::uint8_t, std::string> mmcManufacturersMap{{0x15, "Samsung"},
+                                                                      {0x70, "Kingston"}, // Pure
```
Comment (swichu91) [inline module-platform/rt1051/src/disk_emmc.hpp:54]:
```suggestion
                                                                      {0x70, "Kingston"},
```
I would add better description of manufacturer if possible.

**Review by swichu91 — APPROVED** [review]

File: module-platform/rt1051/src/disk_emmc.cpp [hunk 2]
```diff
@@ -209,4 +211,48 @@ namespace purefs::blkdev
...
+    {
+        const std::uint8_t manufacturerId = mmcCard->cid.manufacturerID;
+        const auto manufacturer           = mmcManufacturersMap.find(manufacturerId);
+        if (manufacturer != mmcManufacturersMap.end()) {
+            return manufacturer->second;
+        }
+        return "unknown (0x" + utils::byteToHex(manufacturerId) + ")";
+    }
+
+    auto disk_emmc::log_emmc_info() const -> void
```
Comment (swichu91) [inline module-platform/rt1051/src/disk_emmc.cpp:224]: How about?
```suggestion
    auto disk_emmc::get_emmc_info_str() const -> std::string
```
Or you can split it into two functions, one for obtaining info structure(you can create custom one containing all the necessary info), and another one for converting it to the human-readable string ;)

Comment (Lefucjusz) [reply module-platform/rt1051/src/disk_emmc.cpp]: Please take a look, I've split it into two functions, one returning custom info structure and second one printing all the required values to the log file

**Pull request merged**